bin/sl-deploy.js

```diff
@@ -81,6 +87,9 @@ if (!serviceName) {
   process.exit(1);
 }
 
+clusterSize = (isNaN(clusterSize) || clusterSize < 1) ?
```

Comment:
removed all the validity check code.  sl-deploy.js just passes the parameter string to deploy as-is -- can be null.